src/clock.h

```diff
@@ -19,4 +19,15 @@ clock_t clock_read();
 clock_t clock_diff(clock_t start, clock_t stop);
 void clock_wait(uint32_t millis);
 
+/**
+ * @brief Compares two times
+ * @details This is a comparison function to the relation between two times
+ * 
+ * @param a The first time
+ * @param b The second time
+ * 
+ * @return Returns -1 if a is bigger, 0 if they are the same, and 1 if b is bigger
```

Comment:
It makes more sense to return -1 if a is *smaller*, and 1 if a is *bigger*. See [strcmp documentation](http://www.cplusplus.com/reference/cstring/strcmp/).